02/app.js

```diff
@@ -1 +1,8 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const linksList = document.querySelectorAll('[data-url]');
+
+linksList.forEach(el => {
```

Comment:
Tutaj już nie trzeba sprawdzać bo to będzie pusta tablica więc po prostu pętla się nie wykona.